api/stunner.yaml

```diff
@@ -77,6 +77,12 @@ paths:
           required: false
           schema:
             type: string
+        - name: public-ip
```

Comment:
Can we still rename this to public-address? Most people requesting this want a domain name here, it'll be terribly confusing to them to name it "public IP"